pkgs/applications/misc/msglint/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation {
+  name = "msglint-1.04";
```

By convention, the package name and the version are separated in two variables. This might has the advantage for the `version` variable to be used within the `src` `url`.

```suggestion
  pname = "msglint";
  version = "1.04";
```

Next to this suggestion, you could add `rec` to the derivation and put the `${version}` within the source's `url`.